### PR TITLE
Upload file option disabled for discharged patients

### LINKED ISSUE #4060: Upload file still working for discharged patients
**Describe the bug**
We can still upload patient files for discharged patients.

**To Reproduce**
Try to upload patient files for any discharged patient

1. Go to https://care.coronasafe.in/facility/005215fa-c20a-4409-aabf-4349dde4504d/patient/0fcb6bf8-99c2-4c22-a259-0e358b060865/consultation/c3514b0d-79e8-4b34-a46b-f28edcc0e73f/files
2. Upload a file

**Expected behavior**
The upload button should be disabled.

**Screenshots**
image

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by Pranshu1902**
Fixes #4060 
Disabled the upload file button for discharged patients.
https://deploy-preview-4061--care-egov-staging.netlify.app/facility/005215fa-c20a-4409-aabf-4349dde4504d/patient/0fcb6bf8-99c2-4c22-a259-0e358b060865/consultation/c3514b0d-79e8-4b34-a46b-f28edcc0e73f/files
image

### CODE REVIEW

**Comment by nihal467** [comment]
LGTM

File: src/Components/Patient/FileUpload.tsx [hunk 1]
```diff
@@ -27,6 +27,7 @@ import Pagination from "../Common/Pagination";
 import { RESULTS_PER_PAGE_LIMIT } from "../../Common/constants";
 import imageCompression from "browser-image-compression";
 import { formatDate } from "../../Utils/utils";
+import { isCallChain } from "typescript";
```
Comment (khavinshankar) [inline src/Components/Patient/FileUpload.tsx]:
```suggestion
```

File: src/Components/Patient/FileUpload.tsx [hunk 2]
```diff
@@ -247,6 +249,10 @@ export const FileUpload = (props: FileUploadProps) => {
         limit: limit,
         offset: offset,
       };
+      const patientDetails = await dispatch(getPatient({ id: patientId }));
+      if (patientDetails.data) {
+        setIsActive(patientDetails.data.is_active);
+      }
```
Comment (khavinshankar) [inline src/Components/Patient/FileUpload.tsx]: we are already calling `getPatient` in line 163, so getting the `is_active` attribute from that would reduce the extra API call

**Review by khavinshankar — CHANGES_REQUESTED** [review]

File: src/Components/Patient/FileUpload.tsx (hunk 1 above)
Comment (Pranshu1902) [reply src/Components/Patient/FileUpload.tsx]: Done

File: src/Components/Patient/FileUpload.tsx (hunk 2 above)
Comment (Pranshu1902) [reply src/Components/Patient/FileUpload.tsx]: Done

**Comment by khavinshankar** [comment]
@nihal467 can you retest this pr?

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**